### ISSUE TITLE
User can change arbitrary team's wallet address

### ISSUE THREAD

**Issue by csanuragjain** [code-423n4/code423n4.com#2188]
## Issue with file
submit-finding.js

## Issue
While submitting issues, a non team member can submit issue for a team and if the team is missing polygon wallet address then can even update the same. This is happening due to includes function usage which allows partial match

## POC
1. User can register with handle "s"
2. This user submits an issue for Team T having member sock, jp, ...
3. This goes to submit-finding.js which checks teamData.members.includes(user)
4. This will return true as "s" is present in sock
5. Now since the team does not have polygon address so updateTeamData(teamData, address) will be called with user provided address

## Recommendation
Instead of using includes, make use of complete word match

**Comment by csanuragjain** [comment]
The includes is used in other files as well, same fix need to be applied there as well

**Comment by captainmangoC4** [comment]
Thanks so much for the time you put into this. I looked into it a little deeper and the `includes()` method is called on the array, not on each element in the array.  While JavaScript `String.prototype.includes` returns true for a partial match, `Array.prototype.includes` matches the entire element (in this case whole word match), so this wouldn't be an issue in this case. I searched the rest of the codebase for `includes` and found it is called on a string in a couple of places, but never on user-entered data (one instance is checking for "json" file extensions and another is checking for specific labels in GH issues). For these reasons, I believe this is not an issue.
